wiki/Schema_Overview.md

```diff
@@ -250,6 +250,28 @@ nodes but lists the downstream, child path instead):
     WHERE name LIKE '%fungi%')
 ```
 
+Return the full taxonomy of every species in the database as a
+semicolon separated list. This query is written for postgres
+and relies on the `string_agg` function. SQLite and MySQL instead
+call the same function `group_concat`.
+
+``` sql
```

Comment:
every other code block in this file has the space, so I was just being consistent.